frame/pablo/src/lib.rs

```diff
@@ -420,6 +440,34 @@ pub mod pallet {
 			)?;
 			Ok(())
 		}
+
+		#[pallet::weight(10_000)]
```

Comment:
We have to benchmark this after merging #827 I think.

Reply:
Yes.